.build.sh

```diff
@@ -1,23 +1,36 @@
 #!/bin/bash
-
+ 
 # echo "==================================== WHICH GCC ==================================== "
 # which gcc
 # ls /usr/local/bin/gcc*
+ 
+# Install prereqs with INSTALL_PREREQS=1 or turn off with INSTALL_PREREQS=0
+export INSTALL_PREREQS=1
+ 
+# Setup tests for APBS with TEST_APBS=1 or turn off with TEST_APBS=0
+export TEST_APBS=0
```

Comment:
This should be `TEST_APBS=1` so that testing is done after the compile. All tests should pass.

Reply:
I implemented a testing step in the GitHub Actions configuration.  I did it there so it could be used by all three systems, and not just Mac and Linux that use `.build.sh`.  Here's the definition of the testing step:
```
      - name: Run tests
        run: |
          ctest -C Release --output-on-failure 
```

Reply:
Also it's useful for things to be running in separate steps in GitHub Actions, if they're logically separable activities.  This is probably more relevant than my above point about a cross-platform implementation.